### ISSUE TITLE
Restart the two failed jobs for 2.17.0

### ISSUE THREAD

**Issue by hmaarrfk** [conda-forge/tensorflow-feedstock#401]
### Comment:

It seems that some builds failed because google-pasta 0.2.0 build 0 was installed 

```
mamba create --name py39 python=3.9 google-pasta=0.2.0=*_0 --channel conda-forge --override-channels
mamba activate py39
pip check
```

```
$ conda list && pip check
# packages in environment at /home/mark/miniforge3/envs/py39:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
bzip2                     1.0.8                h4bc722e_7    conda-forge
ca-certificates           2024.7.4             hbcca054_0    conda-forge
google-pasta              0.2.0              pyh8c360ce_0    conda-forge
ld_impl_linux-64          2.40                 hf3520f5_7    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-ng                 14.1.0               h77fa898_0    conda-forge
libgomp                   14.1.0               h77fa898_0    conda-forge
libnsl                    2.0.1                hd590300_0    conda-forge
libsqlite                 3.46.0               hde9e2c9_0    conda-forge
libuuid                   2.38.1               h0b41bf4_0    conda-forge
libxcrypt                 4.4.36               hd590300_1    conda-forge
libzlib                   1.2.13               h4ab18f5_6    conda-forge
ncurses                   6.5                  h59595ed_0    conda-forge
openssl                   3.3.1                hb9d3cd8_3    conda-forge
pip                       24.2               pyhd8ed1ab_0    conda-forge
python                    3.9.19          h0755675_0_cpython    conda-forge
readline                  8.2                  h8228510_1    conda-forge
setuptools                72.2.0             pyhd8ed1ab_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
tk                        8.6.13          noxft_h4845f30_101    conda-forge
tzdata                    2024a                h0c530f3_0    conda-forge
wheel                     0.44.0             pyhd8ed1ab_0    conda-forge
xz                        5.2.6                h166bdaf_0    conda-forge
google-pasta 0.2.0 is not supported on this platform
```

recreates the issue, though I feel like this was fixed in 
https://github.com/conda-forge/google-pasta-feedstock/pull/3

Is there a way to restart these builds without restarting all builds?

**Comment by xhochy** [comment]
I don't see a rerun button in the GitHub UI for them (anymore). So we probably need to restart the whole matrix.

**Comment by hmaarrfk** [comment]
ok i wanted to make sure I was missing something.

Would you like to merge https://github.com/conda-forge/tensorflow-feedstock/pull/400 instead?

**Comment by njzjz** [comment]
Fixed by #400: https://anaconda.org/conda-forge/tensorflow-base/files
